dependencies.yaml

```diff
@@ -0,0 +1,107 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+ files:
+   all:
+     output: conda
+     matrix:
+       cuda: ["11.5"]
+       arch: [x86_64]
+     includes:
+       - common_build
+       - cpp_build
+       - cudatoolkit
+       - doc
+       - python_build
+ channels:
+   - rapidsai
+   - rapidsai-nightly
+   - dask/label/dev
+   - conda-forge
+   - nvidia
+ dependencies:
+   cudatoolkit:
+     specific:
+       - output_types: conda
+         matrices:
+           - matrix:
+               cuda: "11.5"
+             packages:
+               - cudatoolkit=11.5
+           - matrix:
+               cuda: "11.4"
+             packages:
+               - cudatoolkit=11.4
+           - matrix:
+               cuda: "11.2"
+             packages:
+               - cudatoolkit=11.2
+           - matrix:
+               cuda: "11.0"
+             packages:
+               - cudatoolkit=11.0
```

Comment:
We can remove CUDA 11.0. ([Dropped in 22.10](https://docs.rapids.ai/notices/rsn0021/))
```suggestion
```